vars/kola.groovy

```diff
@@ -125,6 +125,11 @@ def call(params = [:]) {
                 ids += id
                 shwrap("cd ${cosaDir} && cosa kola ${rerun} --output-dir=${outputDir}/${id} --upgrades --build=${buildID} ${archArg} ${platformArgs}")
             } catch(e) {
+                // If we didn't even get logs then let's remove them from the list
+                if (shwrapRc("cd ${cosaDir} && cosa shell -- test -d ${outputDir}/${ids[-1]}") != 0) {
+                    ids.remove(ids.size() - 1)
+                }
```

Comment:
Testing in the console, passing the element itself to `remove` also works. I think it'd be cleaner than using indices. We can hoist the `def id` up to outside the `try` block, and then here we can just use `${outputDir}/${id}` and `ids.remove(id)`.